src/main/resources/site/parts/associatedArticleArchives/associatedArticleArchives.es6

```diff
@@ -0,0 +1,109 @@
+const {
+  data: {
+    forceArray
+  }
+} = __non_webpack_require__('/lib/util')
+const {
+  get
+} = __non_webpack_require__( '/lib/xp/content')
+const {
+  getContent, pageUrl
+} = __non_webpack_require__('/lib/xp/portal')
+const {
+  getPhrases
+} = __non_webpack_require__( '/lib/language')
+const {
+  render
+} = __non_webpack_require__('/lib/thymeleaf')
+const {
+  renderError
+} = __non_webpack_require__('/lib/error/error')
+
+const moment = require('moment/min/moment-with-locales')
+const React4xp = __non_webpack_require__('/lib/enonic/react4xp')
+const view = resolve('./associatedArticleArchives.html')
+
+exports.get = (req) => {
+  try {
+    return renderPart(req)
+  } catch (e) {
+    return renderError(req, 'Error in part: ', e)
+  }
+}
+
+exports.preview = (req) => renderPart(req)
+
+function renderPart(req) {
+  const page = getContent()
+  moment.locale(page.language ? page.language : 'nb')
+  const phrases = getPhrases(page)
+
+  const associatedArticleArchivesHeader = phrases.associatedArticleArchivesHeader
+  const associatedArticleArchivesConfig = page.data.articleArchive ? forceArray(page.data.articleArchive) : []
+  const associatedArticleArchiveLinks = getAssociatedArticleArchiveLinks(associatedArticleArchivesConfig)
+
+  if (!associatedArticleArchivesConfig.length > 0) {
+    if (req.mode === 'edit' && page.type !== `${app.name}:article`) {
+      return {
+        body: render(view, {
+          associatedArticleArchivesHeader
+        })
+      }
+    } else {
+      return {
+        body: null
+      }
+    }
+  }
+
+  const associatedArticleArchiveLinksComponent = new React4xp('Links')
+    .setProps({
+      links: associatedArticleArchiveLinks.map(({
+        title, href
+      }) => {
+        return {
+          children: title,
+          href
+        }
+      })
+    })
+    .uniqueId()
+
+  const body = render(view, {
+    associatedArticleArchivesHeader,
+    associatedArticleArchiveLinksId: associatedArticleArchiveLinksComponent.react4xpId
+  })
+
+  const isOutsideContentStudio = (
+    req.mode === 'live' || req.mode === 'preview'
+  )
+
+  return {
+    body: associatedArticleArchiveLinksComponent.renderBody({
+      body,
+      clientRender: isOutsideContentStudio
+    }),
+    pageContributions: associatedArticleArchiveLinksComponent.renderPageContributions({
+      clientRender: isOutsideContentStudio
+    }),
+    contentType: 'text/html'
+  }
+}
+
+const getAssociatedArticleArchiveLinks = (associatedArticleArchivesConfig) => {
+  if (associatedArticleArchivesConfig.length > 0) {
+    return associatedArticleArchivesConfig.map((articleArchive) => {
+      const articleArchiveContent = get({
```

Comment:
null check this, as the articleArchiveContent might be deleted or unpublished, you'll then have to filter those out from the final array as well